collection-scripts/gather_version

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# load shared functions and data when not sourced
+if [[ -z "$DIR_NAME" ]]; then
+    CALLED=1
+    DIR_NAME=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+    source "${DIR_NAME}/common.sh"
+fi
+
+function ocp_version() {
+    oc_version=$(oc version -o json | jq -r .openshiftVersion)
```

Comment:
I think this is fine... and `openshiftVersion` should refer to the server version.
I did something similar with `oc get clusterversion version -o jsonpath='{.status.desired.version}'` but I think your solution is better.